indico/modules/events/templates/forms/event_creation_form.html

```diff
@@ -0,0 +1,96 @@
+purely client-side and are worded for event creation instead of being generic.
+#}
+<script type="text/html" id="event-creation-protection-messages">
+    <div class="action-box for-form protection-message public-protection-message accept">
+        <div class="section">
+            <div class="icon icon-unlocked"></div>
+            <div class="text">
+                <div class="label">
+                    {% trans %}Public{% endtrans %}
+                </div>
+                <div>
+                    {% trans -%}
+                        The event will be publicly accessible since it is set as public.
+                    {%- endtrans %}
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <div class="action-box for-form protection-message protected-protection-message danger">
+        <div class="section">
+            <div class="icon icon-lock"></div>
+            <div class="text">
+                <div class="label">
+                    {% trans %}Protected{% endtrans %}
+                </div>
+                <div>
+                    {% trans -%}
+                        The event will <strong>only</strong> be accessible by the <strong>managers</strong> of
+                        <strong>parent categories</strong> and users you give access to.
+                    {%- endtrans %}
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <div class="action-box for-form protection-message inheriting-protected-protection-message warning">
+        <div class="section">
+            <div class="icon icon-lock"></div>
+            <div class="text">
+                <div class="label">
+                    {% trans %}Protected{% endtrans %}
+                </div>
+                <div>
+                    {% trans -%}
+                        This event will not be publicly accessible since the category <strong class="js-category-title"></strong> is protected.
+                    {%- endtrans %}
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <div class="action-box for-form protection-message inheriting-public-protection-message accept">
+        <div class="section">
+            <div class="icon icon-unlocked"></div>
+            <div class="text">
+                <div class="label">
+                    {% trans %}Public{% endtrans %}
+                </div>
+                <div>
+                    {% trans -%}
+                        This event will be publicly accessible since the category <strong class="js-category-title"></strong> is not protected.
+                    {%- endtrans %}
+                </div>
+            </div>
+        </div>
+    </div>
+</script>
+
+<script>
+    setupEventCreationDialog({
+        categoryField: $('#{{ form.category.id }}'),
+        protectionModeFields: $('input[name={{ form.protection_mode.name }}][id^={{ form.protection_mode.id }}]'),
+        initialCategory: {{ category | tojson }}
```

Comment:
No rule on it. I tend to add the whitespace usually when there's a function call on the left since it's usually more readable that way. In this case it doesn't really matter...